buildscripts/Makefile.common

```diff
@@ -0,0 +1,28 @@
+# Copyright © 2019 The OpenEBS Authors
```

Comment:
Is it possible to have this file suffixed with something that declares this file as a makefile

Reply:
renamed to common.mk.